content/master/concepts/compositions.md

```diff
@@ -0,0 +1,993 @@
+the Provider.
+
+#### Template a managed resource
+The contents of the 
+{{<hover label="resources" line="6">}}base{{</hover>}} are identical to
+creating a standalone [managed resource]({{<ref "./managed-resources">}}).
+
+This example uses 
+[Upbound's Provider AWS](https://marketplace.upbound.io/providers/upbound/provider-aws/v0.35.0)
+to define a 
+S3 storage {{<hover label="resources" line="8">}}Bucket{{</hover>}} and 
+EC2 compute {{<hover label="resources" line="15">}}Instance{{</hover>}}.
+
+After defining the 
+{{<hover label="resources" line="7">}}apiVersion{{</hover>}} and 
+{{<hover label="resources" line="8">}}kind{{</hover>}}, define the 
+{{<hover label="resources" line="10">}}spec.forProvider{{</hover>}} fields
+defining the settings of the resource.
+
+```yaml {copy-lines="none",label="resources"}
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+spec:
+  resources:
+    - name: StorageBucket
+      base:
+        apiVersion: s3.aws.upbound.io/v1beta1
+        kind: Bucket
+        spec:
+          forProvider:
+            region: "us-east-2"
+    - name: VM
+      base:
+        apiVersion: ec2.aws.upbound.io/v1beta1
+        kind: Instance
+        spec:
+          forProvider:
+            ami: ami-0d9858aa3c6322f73
+            instanceType: t2.micro
+            region: "us-east-2"
+```
+
+When a [Composite Resource]({{<ref "./composite-resources" >}}) uses this
+Composition template, the Composite Resource creates two new managed resources
+with all the provided 
+{{<hover label="resources" line="17">}}spec.forProvider{{</hover>}} settings.
+
+The {{<hover label="resources" line="17">}}spec.forProvider{{</hover>}}
+supports any settings used in a managed resource including
+applying `annotations` and `labels` or using a specific 
+`providerConfigRef`.
+
+{{<hint "note" >}}
+Compositions allow applying a `metadata.name` but ignores it. The
+`metadata.name` field doesn't influence the name
+of the managed resource in Crossplane or the external resource inside the 
+Provider.
+
+Use the `crossplane.io/external-name` annotation on the resource to influence
+the external resource name.
+{{< /hint >}}
+
+#### Template a ProviderConfig
+
+Compositions can define a ProviderConfig like it defines managed resources.
+Generating a ProviderConfig may be useful in providing unique credentials to
+each deployment.
+
+
+```yaml {copy-lines="none"}
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+spec:
+  resources:
+    - name: my-aws-provider-config
+      base:
+        apiVersion: aws.upbound.io/v1beta1
+        kind: ProviderConfig
+        spec:
+          source: Secret
+          secretRef:
+            namespace: crossplane-system
+            name: aws-secret
+            key: creds
+```
+
+
+#### Template another Composite Resource
+
+Compositions can use existing Composite Resources to define more complicated
+templates. 
```

Comment:
"Existing" reads to me like you already have to have _created_ the XR you want to compose. Is it worth finding another name? "Defined" seems correct to me but maybe a bit academic.

Maybe just "other kinds of Composite Resources"?